argo/providers/vai/config_unit_test.go

```diff
@@ -0,0 +1,29 @@
+//go:build unit
+
+package vai
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/sky-uk/kfp-operator/argo/common"
+)
+
+var _ = Context("VAI Config", func() {
+
+	config := VAIProviderConfig{}
+
+	DescribeTable("getMaxConcurrentRunCountOrDefault", func(setValue int64, expectDefault bool) {
+		config.MaxConcurrentRunCount = setValue
+		expectedRes := setValue
+		if expectDefault {
+			expectedRes = 10
+		}
+
+		Expect(config.getMaxConcurrentRunCountOrDefault()).To(Equal(expectedRes))
+
+	},
+		Entry("", int64(0), true),
+		Entry("", -common.RandomInt64(), true),
```

Comment:
does `common.RandomInt64()` always return a positive number? I thought it returned a random sign as well

Reply:
Yes, I double checked and is always a positive int 🤷🏼‍♂️

Reply:
FYI:
```
// Int returns a non-negative pseudo-random int.
func Int() int {
	rng.Lock()
	defer rng.Unlock()
	return rng.rand.Int()
}
```